packages/regl-worldview/src/commands/GLText.js

```diff
@@ -416,9 +458,17 @@ function makeTextCommand() {
 }
 
 export default function GLText(props: Props) {
+  const context = useContext(WorldviewReactContext);
+  const { dimension } = context;
+  const scaleInvariantFontSize = props.scaleInvariantFontSize || DEFAULT_SCALE_INVARIANT_SIZE;
+  const scaleInvariantSize = scaleInvariantFontSize / dimension.height;
```

Comment:
Why exactly is the `scaleInvariantSize` dependent on the height of Worldview? That's not totally obvious to me. Might be worth a comment perhaps?

Reply:
@janpaul123 I agree, it's not obvious. I just added this comment to the code that explains it:

`Compute the actual size for the text object in NDC coordinates (from -1 to 1)
In order to make sure the text is always shown at the same size regardless of the canvas dimensions (as Text does), we need to scale it based on both the desired font size and the current worldview resolution. Otherwise, the text will be displayed at different sizes depending on the worlview canvas dimension.`